sympy/printing/str.py

```diff
@@ -119,11 +122,11 @@ def _print_Dict(self, expr):
 
     def _print_RandomDomain(self, d):
         try:
-            return 'Domain: ' + self._print(d.as_boolean())
+            return 'Domain: (' + self._print(d.as_boolean()) + ')'
```

Comment:
The str printer is supposed use Python syntax, so it can be copied and pasted. But this already isn't following that.